### ISSUE TITLE
Page/Post editor fails to load on a Simple site

### ISSUE THREAD

**Issue by aisajib** [Automattic/themes#8514]
### Quick summary

A WordPress.com Simple site is failing to load its posts and pages in the editor. It's showing the following error: 

```

'sitename'.wordpress.com is currently unable to handle this request. 
HTTP ERROR 500

```

The site's pages also appear blank.

### Steps to reproduce

1. Open the site's posts or pages
2. Click on a post/page title to edit it

### What you expected to happen

1. The post/page content should appear/load in an editor.

### What actually happened

We get the following error from the browser: ![Image](https://github.com/user-attachments/assets/69071fa6-590f-4504-84f5-c7800b4b9bfb)

### Impact

One

### Available workarounds?

No and the platform is unusable

### If the above answer is "Yes...", outline the workaround.

We did not try switching the theme to see what would happen since the Seedlet theme [had a recent issue](https://github.com/Automattic/themes/issues/7937) that's already been fixed. It was later noted that despite having the same theme, the issues are unrelated. 

### Platform (Simple and/or Atomic)

Simple

### Logs or notes

We couldn't reproduce the issue on any other site with the same theme, but on this particular site, you could open any post or page editor and see the browser error shown above. 

Reported in 9154736-zd-a8c
Slack discussion: p1733781425638999-slack-C029FM1EH

**Comment by dsas** [comment]
This is fataling due to a strict type error. 

`substr` is called with a null parameter, ultimately because `seedlet_custom_colors_extra_css` calls `change_color_luminescence` with null rather than with a hex colour [[src](https://github.com/Automattic/themes/blob/f8a5ad0b079ac41bce210fbdc5b68d5f3c44e8b8/seedlet/inc/wpcom-colors-utils.php#L136)]. The customer site affected doesn't define a primary, secondary or tertiary colour.

**Comment by Robertght** [comment]
Marking this as "Triaged" on the One Board.

**Comment by dsas** [comment]
Fixed via wpcom-themes#156508
